styles.css

```diff
@@ -3,53 +3,122 @@
   height: 100vh;
   width: 100vw;
   display: grid;
-  grid-template:
-    'header main main main main'
-    'header main main main main'
-    'header section section section section'
-    'header section section section section'
-    'header section section section section'
-    
+  grid-template-columns: 20% 80%;
```

Comment:
I like this change. It seems much more approachable and cleaner than the template method. Thank you for doing it!